Sources/Paywalls/Events/PaywallEvent.swift

```diff
@@ -36,6 +40,7 @@ extension PaywallEvent {
     public struct Data {
 
         // swiftlint:disable missing_docs
+        public var id: ID?
```

Comment:
Optional to make backwards compatible with previously stored events

Reply:
Let's leave a comment explaining that.